data/human/human missions.txt

```diff
@@ -5416,3 +5416,194 @@ mission "Deal Change 2"
+				goto helpornot
+			label suspicious
+			`    She laughs, "There's no need to be suspicious, my brother and I were just searching for someone who could give us a ride to New China. We received notice that our father's last will is going to be published there. Do you think you could bring us there? We will pay you for your service, of course.`
+			label helpornot
+			choice
+				`	"Of course I can."`
+						goto agreed
+				`	"I'm sorry, but I've got more urgent things to do. You'll need to find someone else."`
+			`	Her smile fades away as you say that, but they both simply nod and walk away out the gates of the spaceport, with both heads looking down.`
+				decline
+			label agreed
+			`    "Fantastic! My name is Sophie, and this is my younger brother," she says, pointing at the man to her side, who goes to shake your hand. "Hi, I'm Liam. We have been looking for a ride for some time now, we've already started worrying that we wouldn't get there in time! You'll have to bring us there quickly."`
+			choice
+				`	"Don't worry, you'll be surprised how fast my ship flies. We'll get there in no time."`
+					goto notime
+				`	"Yes, a lot of captains are quite busy these days."`
+			`	"It seems that way," Sophie agrees. "Everyone feels more tense or uncertain, and there's always more to do and less time to do it."`
+				goto ship
+			label notime
+			`	Liam's voice barely hides his excitement at finding an available captain. "Well, let's see that ship of yours then, and show us if it's as fast as you say it is!"`
+			label ship
+			`	You lead them to your ship, and help them with their bags as they board.`
+				accept
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		log `Helped a pair of siblings named Sophie and Liam by bringing them to New China so they can hear their father's last will.`
+		dialog `When you arrive at New China's spaceport, the siblings prepare to head off to the court where their father's last will will be read out. Just before they leave, Liam turns to you. "Captain <last>, I've transferred <payment> to your account. Would it be okay for you to wait here until we come back? We will need a ride home."`
+		payment 50000
+
+mission "A dead man's path II"
+	name "Following Footprints"
+	description "Bring Sophie and Liam through <waypoints>, before finally bringing them to <destination>. They will pay you another 50,000 credits."
+	passengers 2
+	source
+		planet "New China"
+	destination
+		planet "Prime"
+	waypoint "Terminus"
+	waypoint "Aspidiske"
+	waypoint "Gorvi"
+	waypoint "Enif"
+	to offer
+		has "A dead man's path I: done"
+	on offer
+		conversation
+			`	After about two hours they come back, although unexpectedly they both have broad smiles on their face. You flag them down from your table, and they eagerly take a seat.`
```

Comment:
```suggestion
			`After waiting two hours in the spaceport's bar, Liam and Sophie come back with unexpectedly broad smiles on their faces. You flag them down from your table, and they eagerly take a seat.`
```
I just realised that this wouldn't make sense if you came back to New China later, and I removed the spaceport bar part so I'll have to add it back.